client/src/pages/AddProject.js

```diff
@@ -0,0 +1,251 @@
+import React, { Component } from 'react';
+import { Typography, withStyles, Button, TextField } from '@material-ui/core';
+import OutlinedInput from '@material-ui/core/OutlinedInput';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import validator from 'validator';
+
+import { OutlinedSelect } from '../components/Inputs';
+import UploadImages from '../components/UploadImages';
+import FormValidator from '../helpers/form-validation';
+import { addProject } from '../api/projects';
+import { locations, industries } from '../dummyData/dropDownItems'
+import { withPageContext } from '../components/pageContext';
+
+const styles = {
+    pageContent: {
+        display: 'flex',
+        justifyContent: 'left'
+    },
+    projectPreviewContainer: {
+        display: 'flex',
+        width: '25%',
+        flexDirection: 'column',
+        margin: '0px 20px 0px 10px',
```

Comment:
As a suggestion, You could do something simple like `marginRight: 20` and `marginLeft: 10` instead here